sap-s4hana-soap/2.5/modules/ROOT/pages/sap-s4hana-soap-connector-reference.adoc

```diff
@@ -439,6 +440,62 @@ Defines a configuration for TLS, which can be used from both the client and serv
 ** DAYS | Time unit that qualifies the maxIdleTime attribute. |  |
 |===
 
+[[ListenerSecurityConfig]]
+=== Listener Security Config
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Service Credentials a| <<UsernamePasswordSecurityStrategyConfig>> | Listener client credentials configuration. |  | x
+| Signature Verification Strategy a| One of:
+
+* <<TrustStoreStrategy>>
+* <<DefaultStrategy>> | Strategy used to verify signed SOAP messages. |  |
+|===
+
+[[UsernamePasswordSecurityStrategyConfig]]
+=== Username Password Security Strategy Config
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Clients a| Array of <<UsernamePasswordEntry>> | List of authorized usernames and passwords. |  |
+|===
+
+[[UsernamePasswordEntry]]
+=== Username Password Entry
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Username a| String | Client's username. |  | x
+| Password a| String | Client's password. |  | x
+|===
+
+[[DefaultStrategy]]
+=== Default Strategy
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| CACerts Password a| String | Password for the default Java cacerts truststore. This store is located in `<JAVA_HOME>/lib/security/cacerts`. | `changeit` |
+|===
+
+[[TrustStoreStrategy]]
+=== Trust Store Strategy
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Trust store path a| String | Path to your truststore. This is either an absolute path or a path relative to your /resources folder. |  | x
```

Comment:
Is the / meant to represent root or something, or is it /resources? 
If it's not either one, then you can delete the /.